test/e2e/util/util.go

```diff
@@ -148,6 +148,8 @@ func WaitForNReadyNodes(t *testing.T, ctx context.Context, client crclient.Clien
 	switch platform {
 	case hyperv1.PowerVSPlatform:
 		waitTimeout = 60 * time.Minute
+	case hyperv1.KubevirtPlatform:
+		waitTimeout = 60 * time.Minute
```

Comment:
looks like with RHCOS 9.2 the boot time is longer, 30 minutes are not enough for the VM to fully boot twice. extending to 60.